core/players-handbook-2024/races/race-goliath.xml

```diff
@@ -0,0 +1,149 @@
+		</update>
+	</info>
+	
+	<element name="Goliath" type="Race" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACE_GOLIATH">
+		<description>
+			<p>Towering over most folk, goliaths are distant de­scendants of giants. Each goliath bears the favors of the first giants—favors that manifest in various supernatural boons, including the ability to quickly grow and temporarily approach the height of goli­aths’ gigantic kin.</p>
+			<p class="indent">Goliaths have physical characteristics that are reminiscent of the giants in their family lines. For example, some goliaths look like stone giants, while others resemble fire giants. Whatever giants they count as kin, goliaths have forged their own path in the multiverse—unencumbered by the internecine conflicts that have ravaged giantkind for ages—and seek heights above those reached by their ancestors.</p>
+			
+			<h4>Goliath Traits</h4>
+			<p style="margin-top:15px"><strong><em>Creature Type:</em></strong> Humanoid</p>
+			<p><strong><em>Size:</em></strong> Medium (about 7–8 feet tall)</p>
+			<p><strong><em>Speed:</em></strong> 35 feet</p>
+			
+			<p>As a Goliath, you have these special traits.</p>
+			<p class="indent"><strong><em>Giant Ancestry.</em></strong> You are descended from Giants. Choose one of the following benefits—a supernatural boon from your ancestry; you can use the chosen benefit a number of times equal to your Proficiency Bonus, and you regain all expended uses when you finish a Long Rest:</p>
+			<ul class="unstyled">
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Cloud’s Jaunt (Cloud Giant).</strong> As a Bonus Action, you magically teleport up to 30 feet to an unoccupied space you can see.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Fire’s Burn (Fire Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Frost’s Chill (Frost Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ld6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Hill’s Tumble (Hill Giant).</strong> When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Stone’s Endurance (Stone Giant).</strong> When you take damage, you can take a Reaction to roll ldl2. Add your Constitution modifier to the number rolled and reduce the damage by that total.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Storm’s Thunder (Storm Giant).</strong> When you take damage from a creature within 60 feet of you, you can take a Reaction to deal ld8 Thunder damage to that creature.</li>
```

Comment:
seems like anywhere 1d# was copied from your source, it translated it into letters instead of numbers

```suggestion
				<li style="margin-left:1em;text-indent:-1em;"><strong>Cloud’s Jaunt (Cloud Giant).</strong> As a Bonus Action, you magically teleport up to 30 feet to an unoccupied space you can see.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Fire’s Burn (Fire Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal 1d10 Fire damage to that target.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Frost’s Chill (Frost Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal 1d6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Hill’s Tumble (Hill Giant).</strong> When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Stone’s Endurance (Stone Giant).</strong> When you take damage, you can take a Reaction to roll 1d12. Add your Constitution modifier to the number rolled and reduce the damage by that total.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Storm’s Thunder (Storm Giant).</strong> When you take damage from a creature within 60 feet of you, you can take a Reaction to deal 1d8 Thunder damage to that creature.</li>
```